examples/sdk-app-example/build.gradle

```diff
@@ -92,73 +100,9 @@ dependencies {
     }
 }
 
+apply from: 'checkstyle.gradle'
 apply plugin: 'com.bugsnag.android.gradle'
-bugsnag {
-    ndk true
```

Comment:
Is the bugsnag config being intentionally removed here?

Reply:
The plugin automatically detects whether a project uses the NDK so this isn't strictly required.